### ISSUE TITLE
p-toggleSwitch Accessibility Issues

### ISSUE THREAD

**Issue by srujana10499** [primefaces/primeng#18766]
### Describe the bug

The p-toggleSwitch component with checkboxes has accessibility violations when running the IBM Equal Accessibility Checker. The two issues are: Each form control must have an associated label and UI component must have aria attributes.

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-dltq5smk

### Environment

stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 'Form control with "switch" role has no associated label' and 'The ARIA attributes "aria-checked" are not valid for the element <input> with ARIA role "switch"'

### Expected behavior

There are no accessibility violations for the p-toggleSwitch component when toggleSwitch is  checked/used
